packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,67 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    userIsAdmin: ->
+      organization = Organizations.findOne(Template.instance().data.organizationId)
+      organization.userIsAdmin(Meteor.userId())
+
+    hideButtonsForProfile: (profileId) ->
```

Comment:
I don't think this is a very good name - it's not clear what it's doing or what its purpose is.